### PR TITLE
fix: award guild

### PR DESCRIPTION
**Pull request by Chicken**
only accept awards from the default guild

### CODE REVIEW

**Comment by KommentatorForAll** [comment]
I mean... it doesn't break anything and it looks cool......

**Comment by Chicken** [comment]
It does result in unresolved emojis in notification messages and looks weird on messages if they have awards from multiple guilds

**Review by KommentatorForAll — APPROVED** [review]

**Review by WilsontheWolf — APPROVED** [review]
You're no fun

File: src/dto/ResponseDto.js [hunk 1]
```diff
@@ -3,7 +3,6 @@ export default class ResponseDto {
     /**
      * Create new auto response data.
      * @param {String[]} trigger - The regex to trigger.
-     * @param {boolean} isRegex - If the trigger is a regular expression
```
Comment (WilsontheWolf) [inline src/dto/ResponseDto.js:6]: What

**Review by WilsontheWolf — CHANGES_REQUESTED** [review]

**Review by WilsontheWolf — APPROVED** [review]
You're still no fun

**Pull request merged**